src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -158,6 +159,7 @@ template <typename TFunc, typename... Args>
 [[nodiscard]] mgp_error WrapExceptions(TFunc &&func, Args &&...args) noexcept {
   static_assert(sizeof...(args) <= 1, "WrapExceptions should have only one or zero parameter!");
   try {
+    memgraph::utils::MemoryTracker::OutOfMemoryExceptionEnabler oom_enabler;
```

Comment:
I suppose you've put it here because every method uses this as a wrapper. Is there something which doesn't use WrapExceptions?

Reply:
Yes, that is true. All methods should use WrapExceptions, I will double check